### DISCUSSION TITLE
Screen rotation doesn't rotate video to landscape in full screen in android simulator.

### DISCUSSION THREAD

**Discussion by arapocket** [TheWidlarzGroup/react-native-video#4210] [Q&A] [answered]
### What are you trying to do?

Screen rotation doesn't rotate video to landscape in full screen in android simulator.

Is this a bug? Or some config I'm missing for android. It works fine for iOS.

### Platforms

Android, iOS

**Comment by freeboub** [comment]
or a bug in ios :)
BTW you can force orientation with https://github.com/wonday/react-native-orientation-locker

**Reply by arapocket** [reply]
I was just curious if that is the default behavior for android. Don't want to resort to using a library if I don't need to.

**Reply by freeboub** [reply, marked as answer]
> I was just curious if that is the default behavior for android. Don't want to resort to using a library if I don't need to.

It is an incoherent behavior you are reporting here, but the aim of react-native-video is not manage orientation, so in my point of view it's more an ios issue. (even if full screen in portrait mode doesn't make sense for films, but it make sens of portrait videos ...)

**Reply by arapocket** [reply]
Fair enough. Appreciate the response.

**Reply by arapocket** [reply]
Goes without saying, thank you for your work on this project!
